pkg/reconciler/mtbroker/trigger/controller_test.go

```diff
@@ -19,21 +19,33 @@ package mttrigger
 import (
 	"testing"
 
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"knative.dev/pkg/configmap"
 	. "knative.dev/pkg/reconciler/testing"
+	"knative.dev/pkg/system"
 
 	// Fake injection informers
 	_ "knative.dev/eventing/pkg/client/injection/informers/eventing/v1/broker/fake"
 	_ "knative.dev/eventing/pkg/client/injection/informers/eventing/v1/trigger/fake"
 	_ "knative.dev/eventing/pkg/client/injection/informers/messaging/v1/subscription/fake"
+	"knative.dev/eventing/pkg/reconciler/mtbroker"
```

Comment:
This import should be relocated, it is not a "Fake injection informer" so future readers will be confused.